### DISCUSSION TITLE
need help with parameters annotator resolution and low/high threshold

### DISCUSSION THREAD

**Discussion by RoyDingZF** [Mikubill/sd-webui-controlnet#466] [Q&A]
what does the annotator resolution and low/high threshold do in canny model?

**Comment by Petas3** [comment]
You can just try to play with it using the 'Preview annotator result' and see the results, it has some effect, but not so much. Some detail can be 'lost' if this is not set correctly. Depends on what you want, the canny algo is pretty straightforward compared to the others.
